src/platform/visualc/ntddcdrm.h

```diff
@@ -1,320 +0,0 @@
-/*
- * ntddcdrm.h
- *
- * CDROM IOCTL interface.
- *
- * This file is part of the w32api package.
- *
- * Contributors:
- *   Created by Casper S. Hornstrup <chorns@users.sourceforge.net>
- *
```

Comment:
Interesting.. found that this is now part of MinGW, and more revised versions are out there.  I wonder if this was on anyone's radar to keep it up to date; or if they were aware of it, but choose not to?   In anycase, glad all these OS-specific hooks are being tossed!

More general comment:
A document that lists externally-maintained files like this (same w/ the single-header audio decoders), would be handy for future devs to keep them up-to-date with upstream.

Reply:
Hmm, yeah - so far (aside from `decoders`) I knew about files from MAME, FreeDOS and gui_tk (which seems unused); simple grep inside src reveals few more files, but there are some false positives as well:

```
$ git grep Copyright | grep -v DOSBox
dos/dos_codepages.h: *  Copyright (C)  Henrique Peron
dos/dos_codepages.h:   Copyright (C) 1993-1996 by kostis@acm.org (Kosta Kostis) */
dos/dos_codepages.h:   Copyright (c) 1996-2002 Markus Oberhumer & Laszlo Molnar
dos/dos_keyboard_layout_data.h: *  Copyright (C)  Henrique Peron
dos/dos_keyboard_layout_data.h:   Copyright (C) 2004 by Aitor SANTAMARIA_MERINO */
dos/dos_mscdex.cpp:		case 0x1502:	/* Get Copyright filename */
hardware/mame/fmopl.cpp:** Copyright Jarek Burczynski (bujar at mame dot net)
hardware/mame/fmopl.cpp:** Copyright Tatsuyuki Satoh , MultiArcadeMachineEmulator development
hardware/mame/ymf262.cpp:** Copyright Jarek Burczynski
hardware/opl.cpp: * Copyright (C) 1998-2001 Ken Silverman
hardware/opl.h: * Copyright (C) 1998-2001 Ken Silverman
hardware/sblaster.cpp:	case 0xe3:	/* DSP Copyright */
libs/decoders/SDL_sound.c: * Copyright (C) 2001  Ryan C. Gordon.
libs/decoders/SDL_sound.h: * Copyright (C) 2001  Ryan C. Gordon.
libs/decoders/SDL_sound_internal.h: * Copyright (C) 2001  Ryan C. Gordon.
libs/decoders/docs/LICENSE.txt: Copyright (C) 1991, 1999 Free Software Foundation, Inc.
libs/decoders/docs/LICENSE.txt:    Copyright (C) <year>  <name of author>
libs/decoders/dr_flac.h:Copyright 2018 David Reid
libs/decoders/dr_mp3.h:Copyright 2018 David Reid
libs/decoders/dr_wav.h:Copyright 2018 David Reid
libs/decoders/stb.h:Copyright (c) 2017 Sean Barrett
libs/decoders/stb_vorbis.h:Copyright (c) 2017 Sean Barrett
libs/decoders/xxh3.h:   Copyright (C) 2019-present, Yann Collet.
libs/decoders/xxhash.c:*  Copyright (C) 2012-2016, Yann Collet
libs/decoders/xxhash.h:   Copyright (C) 2012-2016, Yann Collet.
libs/gui_tk/gui_tk.cpp: *  Copyright (C) 2005-2007 Jörg Walter
libs/gui_tk/gui_tk.h: * Copyright (C) 2005-2007 Jörg Walter
libs/zmbv/drvproc.cpp:// Copyright (c) 1993 Microsoft Corporation.
libs/zmbv/zmbv_vfw.cpp:// Copyright (c) 1993 Microsoft Corporation.
```